packages/block-editor/src/components/block-heading-level-dropdown/stories/index.story.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelDropdown from '../';
+
+const meta = {
+	title: 'BlockEditor/HeadingLevelDropdown',
+	component: HeadingLevelDropdown,
+	argTypes: {
+		value: {
+			control: { type: null },
+			description: 'The currently selected heading level.',
+		},
+		options: {
+			control: 'array',
```

Comment:
```suggestion
			control: 'check',
			options: [ 1, 2, 3, 4, 5, 6 ],
```

The only values ​​allowed are numbers from 1 to 6, so like with [textAlignmentControl](https://github.com/WordPress/gutenberg/blob/7cf1ceddbef6ea3802014c43ea16d193febcb6d1/packages/block-editor/src/components/text-alignment-control/stories/index.story.js#L19), I personally think checkboxes are a better choice:

![image](https://github.com/user-attachments/assets/e06c9327-4009-48ee-80a5-8313b6743af6)